src/main/resources/application.properties

```diff
@@ -7,7 +7,8 @@ spring.datasource.password=rNT5Vn[j(>R2r6.UXcJwtv];`RWewYH`
 # hibernate.X
 
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
-spring.jpa.hibernate.ddl-auto=none
+spring.jpa.hibernate.ddl-auto=validate
+  #none
```

Comment:
Please don't change this in your pull request. It should be done by @vladvorontsov in his task